### DISCUSSION TITLE
jQuery version and using with WordPress

### DISCUSSION THREAD

**Discussion by codeoz-com** [6pac/SlickGrid#649] [Q&A] [answered]
Hello, I am looking at using SlickGrid with a new WordPress(WooCommerce) plugin we are developing. I see WordPress is still using v1.x of jQuery, which works fine with SlickGrid. But, WordPress is apparently moving to v3 of jQuery. Does anyone know if SlickGrid will still work as a WordPress plugin if WordPress is updated to v3 jQuery?

**Comment by 6pac** [comment, marked as answer]
Hi @codeoz-com, the latest SlickGrid from the @6pac or @ghiscoding  repos should work fine with jQuery 3.x.

**Reply by codeoz-com** [reply]
Thanks, I realized that WordPress has actually been upgraded to jQuery 3.x, and it works fine.
